route.go

```diff
@@ -3,33 +3,130 @@ package netlink
 import (
 	"fmt"
 	"net"
+	"strings"
 	"syscall"
 )
 
-// Scope is an enum representing a route scope.
-type Scope uint8
-
-const (
-	SCOPE_UNIVERSE Scope = syscall.RT_SCOPE_UNIVERSE
-	SCOPE_SITE     Scope = syscall.RT_SCOPE_SITE
-	SCOPE_LINK     Scope = syscall.RT_SCOPE_LINK
-	SCOPE_HOST     Scope = syscall.RT_SCOPE_HOST
-	SCOPE_NOWHERE  Scope = syscall.RT_SCOPE_NOWHERE
-)
```

Comment:
My general hope is that people could use the library without importing syscall. I'd be ok with a pr to remove the shadowing if people think it is a good idea, but we should do everything at once if we do it.